pyaedt/edb_core/edb_data/sources.py

```diff
@@ -274,34 +275,52 @@ def net(self, value):
     def net_name(self):
         return self._edb_pin_group.GetNet().GetName()
 
+    @property
+    def terminal(self):
+        """Get terminal."""
```

Comment:
```suggestion
        """Terminal."""
```
Properties are described as nouns.